types/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for Apache Cordova Android platform
+// Project: https://github.com/apache/cordova-platform
+// Definitions by: Microsoft Open Technologies Inc <http://msopentech.com>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// 
+// Copyright (c) Microsoft Open Technologies Inc
+// Licensed under the MIT license.
+
+interface Navigator {
+    /** This plugin displays and hides a splash screen during application launch. */
+    splashscreen: {
+        /** Dismiss the splash screen. */
+        hide(): void;
+        /** Displays the splash screen. */
+        show(): void;
```

Comment:
I am not sure what should be done in this case.

The `.show` method does not exist in Android. It had existed in the old splashscreen implementation, but because of the Android 12 Splash Screen API implementation pattern, this is no longer valid.

I believe it is still supported in iOS.

If the user was developing an app for both iOS and Android, I suspect the `.show` definition to appear because of the iOS's `d.ts`. 

If only Android, I believe the `.show` is not visible.

Reply:
Yep, agreed, I can remove this from the android typings.